rules/autoconsent/pubtech.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "pubtech",
+    "prehideSelectors": ["#pubtech-cmp"],
+    "detectCmp": [
+      { 
+        "exists": "#pubtech-cmp" 
+      }
+    ],
+    "detectPopup": [
+      { 
+        "visible": "#pubtech-cmp" 
```

Comment:
this doesn't work because it returns true even when popup is not visible. Try checking the visibility of the accept button?

Reply:
Changes made to target elements of the opened banner